compiler/rustc_middle/src/ty/context.rs

```diff
@@ -2241,7 +2242,7 @@ impl<'tcx> TyCtxt<'tcx> {
     /// to identify which traits may define a given associated type to help avoid cycle errors.
     /// Returns a `DefId` iterator.
     fn super_traits_of(self, trait_def_id: DefId) -> impl Iterator<Item = DefId> + 'tcx {
-        let mut set = FxHashSet::default();
+        let mut set = StableSet::default();
```

Comment:
This doesn't look like it needs to be a `StableSet`. Did you get an error message for it?

Reply:
Done in https://github.com/rust-lang/rust/pull/94188/commits/95ed0262b63ca8d8760970ca04af9d5146c067af, but see compilation log.